images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -199,12 +199,18 @@ $markdown += New-MDList -Style Unordered -Lines (@(
 )
 
 $markdown += New-MDHeader "Browsers and Drivers" -Level 3
-$markdown += New-MDList -Style Unordered -Lines @(
+
+$browsersAndDriversList = @(
     (Get-ChromeVersion),
     (Get-ChromeDriverVersion),
     (Get-FirefoxVersion),
     (Get-GeckodriverVersion)
 )
+if (Test-IsUbuntu20) {
+    $browsersAndDriversList = @(Get-ChromiumVersion) + $browsersAndDriversList
+}
+
+$markdown += New-MDList -Style Unordered -Lines ($browsersAndDriversList | Sort-Object)
```

Comment:
Why do we need `Sort-Object` here? I guess it will break order

Reply:
Fixed